projects/idea-hub-smart-contract-solidity/contracts/IdeaHub.sol

```diff
@@ -0,0 +1,288 @@
+pragma solidity >=0.6 <0.7;
+
+import "./Authenticated.sol";
+import "./IERC20.sol";
+import "./IIdeaHub.sol";
+import "./IdeaHubSigning.sol";
+
+/**
+/* @dev IdeaHub contract. After deploying the contract immediately set the params using
+/* function setPlatformParams and setSigningParams
+*/
+contract IdeaHub is Authenticated, IIdeaHub, IdeaHubSigning {
```

Comment:
Please add documentation for all the public methods

Reply:
Public methods are documented in the Contract Interface, I've added reference to the method as documentation.